src/FetchClient.ts

```diff
@@ -82,3 +89,8 @@ async function fetchResponseBody(response: Response) {
     ? response.json()
     : response.text();
 }
+
+function generateTraceidHeaderValue() {
+  const traceId = uuid().replace(/-/g, '');
```

Comment:
Wow, I'm curious how you came across that comment! This is definitely not performance sensitive code, but I will make the change anyway, since it's a better implementation 😄

Reply:
> Wow, I'm curious how you came across that comment! This is definitely not performance sensitive code, but I will make the change anyway, since it's a better implementation 😄

😆 It all came from this thought: _Hmm, I'm surprised that `uuid` doesn't have a means for configuration. _ As it turns out, `uuid` doesn't have a means for configuration because string replacement is sufficient for the majority of use cases